packages/presets/client/src/index.ts

```diff
@@ -160,8 +162,10 @@ export const preset: Types.OutputPreset<ClientPresetConfig> = {
     const pluginMap = {
       ...options.pluginMap,
       [`add`]: addPlugin,
-      [`typescript`]: typescriptPlugin,
+      // TODO: Remove this
       [`typescript-operations`]: typescriptOperationPlugin,
+      // END TODO -----------------
+      [`typescript-ast-client-preset-only`]: typescriptASTPoweredPlugin,
```

Comment:
```suggestion
      'typescript-ast-client-preset-only': typescriptASTPoweredPlugin,
```

same for others in this object